spec/jobs/pull_hotlinked_images_spec.rb

```diff
@@ -29,4 +29,42 @@
     expect(post.raw).to match(/^<img src='\/uploads/)
   end
 
+  describe 'onebox' do
+
+    Onebox::Engine::WikimediaOnebox.class_eval do
```

Comment:
Those examples in the onebox gem source are either creating new classes with inheritance or including the `Onebox::Engine` module. Monkey patching here means that you're also leaking state because you can't really undo the monkey patch.